src/Windows/Avalonia.Win32/Win32PlatformOptions.cs

```diff
@@ -23,6 +23,7 @@ public enum Win32RenderingMode
     /// Avalonia would try to use native Widows OpenGL with GPU rendering.
     /// </summary>
     Wgl = 3,
+
```

Comment:
nit

Reply:
The spacing in that enum was inconsistent. I added the new line to match the rest of the enum